Dockerfile

```diff
@@ -5,5 +5,6 @@ RUN ./gradlew --no-daemon bootJar --warning-mode all
 
 RUN find . -type f -name example-*.jar -not -iname '*javadoc*' -not -iname '*sources*' -exec cp '{}' '/app.jar' ';'
 ENV JDK_JAVA_OPTIONS "-XX:MaxRAMPercentage=90.0"
-ENTRYPOINT ["java", "--add-opens", "java.base/java.lang=ALL-UNNAMED", "-jar", "/app.jar"]
+COPY jacocoJars/jacocoagent.jar ./jacocoagent.jar
+ENTRYPOINT ["java", "--add-opens", "java.base/java.lang=ALL-UNNAMED", "-javaagent:jacocoJars/jacocoagent.jar=destfile=jacocoOutput/jacoco-it.exec,output=file", "-jar", "/app.jar"]
```

Comment:
double check the javaagent location inside the container, the copy step copies it into this directory